Core/Object Arts/Dolphin/Base/BlockClosure.cls

```diff
@@ -549,6 +549,33 @@ stackTempCount: anInteger
 	info := (info bitAnd: ##((16rFF << 15) bitInvert)) 
 				bitOr: ((anInteger bitAnd: 16rFF) bitShift: 15)!
 
+storeOn: aStream
+	"Append, to aStream, a <String> whose characters can recreate the receiver."
+
+	| m block class selector string |
+	m := self method.
+	(m notNil and: [m isExpression]) ifTrue: [	"We can print the source"
```

Comment:
In existing views, the literal array creates a BlockClosure that names a source method but may be completely different so there isn't really any source anywhere.